booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java

```diff
@@ -294,43 +294,6 @@ private static Consumer<MultiAttributeValidator> allDg(Consumer<MultiAttributeVa
     }
   );
 
-
```

Comment:
please note this validation is nor more valid in booking and hence removed after confirmation from Flavia and Henrik

Reply:
Yeah looks to be the case. I will reinstate it.